connectors/servicenow/README.md

```diff
@@ -1,86 +1,7 @@
 # ServiceNow Connector
 
-## Building
+The ServiceNow connector presents cards inviting the user to approve or reject open requests. It does this based on request numbers passed as tokens in the card request.
```

Comment:
I forgot to mention that the user's email is also passed in: https://github.com/vmware/connectors-workspace-one/blob/master/connectors/servicenow/src/main/resources/static/discovery/metadata.hal#L6

Please change this from `... based on request numbers passed as tokens in the card request` to `... based on request numbers passed as tokens and the user's email address in the card request`.